src/app/template-library/template-library.component.ts

```diff
@@ -0,0 +1,47 @@
+import { Component, OnInit } from '@angular/core';
+import {CategorisedTemplates, Category, Template, TemplateLibraryService} from './template-library.service';
+
+@Component({
+  selector: 'app-template-library',
+  templateUrl: './template-library.component.html',
+  styleUrls: ['./template-library.component.scss'],
+})
+export class TemplateLibraryComponent implements OnInit {
+
+  constructor(private service: TemplateLibraryService) { }
+
+  loadingExps = false;
```

Comment:
`loadingTemplates`

Reply:
This `loadingTemplates` is used to display skeleton screen before we load the actual data.
That skeleton screen part is missing in the HTML.
`ion-skeleton-text` this is the component we use for the skeleton element